perl/bin/gherkin-generate-ast

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use lib 'lib';
+
+use Cpanel::JSON::XS;
```

Comment:
I'm getting this error:

```
Can't locate Cpanel/JSON/XS.pm in @INC (you may need to install the Cpanel::JSON::XS module) (@INC contains: lib /Library/Perl/5.18/darwin-thread-multi-2level /Library/Perl/5.18 /Network/Library/Perl/5.18/darwin-thread-multi-2level /Network/Library/Perl/5.18 /Library/Perl/Updates/5.18.2/darwin-thread-multi-2level /Library/Perl/Updates/5.18.2 /System/Library/Perl/5.18/darwin-thread-multi-2level /System/Library/Perl/5.18 /System/Library/Perl/Extras/5.18/darwin-thread-multi-2level /System/Library/Perl/Extras/5.18 .) at bin/gherkin-generate-ast line 7.
BEGIN failed--compilation aborted at bin/gherkin-generate-ast line 7.
```

This is because the `.built` target depends on `.cpanfile_dependencies` *after* running this script - and that's too late.